GettingStarted.md

```diff
@@ -0,0 +1,167 @@
+# Getting started with TEE development
+The instructions below will help you get your development environment configured so that you can build, test and debug Team Explorer Everywhere.
+
+## Clone and configure the repository
+Use the Git tool of your choice to clone the repository into a local path.
+For example, you could use the command-line client from a console/terminal window:
+```
+git clone https://github.com/Microsoft/team-explorer-everywhere.git
+```
+
+We keep LF line-ending on the server. Please set the `core.safecrlf` git config property to `true`.
+```
+git config --local core.safecrlf true
+```
+
+## Install development tools
+
+### Install Java
+
+Two versions of the Java Development Kit are needed: we use Java SE 6 as the minimal supported Java execution environment and to build the release kit, as well as Java SE 8 for running Eclipse.
+
+1. Download and install Java SE 6:
+  * [Windows and Linux: Java Development Kit 6u45](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase6-419409.html#jdk-6u45-oth-JPR)
+  * [Mac: Apple Java 1.6.0_65, a.k.a. Java for OS X 2015-001](https://support.apple.com/kb/DL1572?locale=en_US)
+1. Ensure the `JAVA_HOME` environment variable is persistently configured to point to the JDK folder.
+1. Ensure the `bin` sub-folder under `JAVA_HOME` is persistently added to the `PATH` environment variable.
+1. Download and install Java SE 8 JDK from [Java SE Downloads](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+
+### Install Ant
+
+1. If you do not already have it, download and extract Apache Ant(TM) version 1.9.6 from [Ant Binary Distributions](http://ant.apache.org/bindownload.cgi).
+1. Ensure the path of the Ant `bin` sub-folder is persistently added to the `PATH` environment variable.
+
+### Install Eclipse
+
+Two versions of Eclipse are needed: we use Eclipse 3.5.2 (Galileo) as the Target Environment, which is the minimum supported Eclipse version (used for building the release) and Eclipse 4.5.2 (Mars.2) as a dedicated team-explorer-everywhere Development Environment.  Using a dedicated Eclipse installation and workspace for development makes it less disruptive to configure Eclipse-wide preferences.
```

Comment:
We use Eclipse 3.5.2 not as the Target Environment, but as a plugins build tool. The statement about "Target Environment" sounds confusing.